src/main/java/org/opentestsystem/ap/ims/repository/SavedSearchRepository.java

```diff
@@ -0,0 +1,16 @@
+package org.opentestsystem.ap.ims.repository;
+
+import org.opentestsystem.ap.ims.entity.SavedSearch;
+import org.opentestsystem.ap.ims.entity.SavedSearch.SavedSearchIdentity;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+public interface SavedSearchRepository extends JpaRepository<SavedSearch, SavedSearchIdentity> {
+
+    List<SavedSearch> findByIdUserName(String userName, Sort orderBy);
+
+    List<SavedSearch> findBySharedSavedSearchesIdSharedWithUserName(String sharedWithUserName);
```

Comment:
Spring Data is pretty cool.  We are able to define method names according to a specific format and Spring Data implements it.  For example "findByIdUserName" is saying "query where savedSearch.id.userName is equal to..."